plugins/catalog-backend-module-azure-blobstorage/README.md

```diff
@@ -0,0 +1,10 @@
+# catalog-backend-module-azure-blobstorage
```

Comment:
Hm. This entire module - any reason it's just not put inside the existing catalog-backend-module-azure? Would probably make it easier to find and use, and have little-to-no drawbacks right?

Reply:
The existing `catalog-backend-module-azure` is specifically for "Azure Devops".  Azure Devops is more like Github & Githlab for CI/CD.  Azure however is the cloud more like GCP and AWS.  (Yay for Microsoft naming).  The two are separate products you can use independently.  (e.g. you can use Azure without Azure Devops and you can also use Azure Devops to deploy to GCP and AWS without the Azure cloud.  

I think it makes sense to keep the two separate, however since we've used `azure` to refer to `azure-devops`, is may be more trouble than it's worth to treat them separately.  (We'd have to rename `catalog-backend-module-azure` to `catalog-backend-module-azure-devops`, and then repurpose `catalog-backend-module-azure` for the azure cloud integrations which sounds like a recipe for confusion & support issues).  

If we do decide to bundle the azure plugins together, it might also make sense to bundle `plugin-catalog-backend-module-msgraph` into the same package for consistency.

Reply:
So if I read that right, might it be ok to put this code in `catalog-backend-module-azure` for now then with a future intent of breaking out the old contents into `catalog-backend-module-azure-devops` and deprecating the symbols in their old place?